tracing-appender/src/lib.rs

```diff
@@ -1,48 +1,101 @@
+//! Writers for logging events and spans
+//!
+//! # Overview
+//!
+//! [`tracing`][tracing] is a framework for instrumenting Rust programs to collect
+//! structured, event-based diagnostic information. This crate provides the ability
+//! for `tracing` events and spans to be recorded in a non-blocking manner by using a
+//! dedicated logging thread. It also provides a [`RollingFileAppender`][file_appender]
+//! that can be used with or without the non-blocking writer.
```

Comment:
If you accept my suggest changes, apply them to the lib.rs as well?

This might also require an MSRV bump (cc: @hawkw), but something like https://blog.guillaume-gomez.fr/articles/2020-03-07+cfg%28doctest%29+is+stable+and+you+should+use+it should reduce the repetition between the README.md and top-level module documentation.

Reply:
Applied to lib.rs too